catalog/core/catalog-core-directorymonitor/pom.xml

```diff
@@ -125,11 +138,16 @@
                             platform-util,
                             platform-util-unavailableurls,
                             failsafe,
-                            hazelcast
+                            hazelcast,
+                            sardine,
+                            httpclient,
```

Comment:
looks like `httpclient` and `httpcore` are osgi bundles, any reason why we're embedding instead of using their bundles?